### PR TITLE
update bx-python

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ('python-lzo', '1.12', {
+        'modulename': 'lzo',
+        'checksums': ['97a8e46825e8f1abd84c2a3372bc09adae9745a5be5d3af2692cd850dac35345'],
+    }),
+    (name, version, {
+        'modulename': 'bx',
+        'checksums': ['ad0808ab19c007e8beebadc31827e0d7560ac0e935f1100fb8cc93607400bb47'],
+    }),
+]
+
```
Comment (smoors) [inline easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb:35]: @fizwit please add here:
```suggestion

sanity_pip_check = True

```

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('LZO', '2.10'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+]
+
+prebuildopts = "export CPATH=$EBROOTLZO/include/lzo:$CPATH && "
```
Comment (smoors) [inline easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb:19]: @fizwit this is a pre-existing issue: this line does not affect extensions, so it does not do anything. see below for a proper fix.

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('python-lzo', '1.12', {
+        'modulename': 'lzo',
```
Comment (smoors) [inline easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb:27]:
```suggestion
        'modulename': 'lzo',
        'preinstallopts': 'export PREFIX=$EBROOTLZO && ',
```

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb:35]: sanity_pip changes made.

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb (hunk 3 above)
Comment (fizwit) [reply easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb:26]: preinstallopts fixed

File: easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (fizwit) [reply easybuild/easyconfigs/b/bx-python/bx-python-0.8.8-foss-2019b-Python-3.7.4.eb]: prebuildopts relocated to exts_list

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/e131459f340af4ad2976480fc8cce124 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @fizwit!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3502.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7302P 16-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/4758f7f3b80e68b3e6cc438421623d4d for a full test report.

**Pull request merged**